LBDependency/.gitignore

```diff
@@ -1,8 +0,0 @@
-.DS_Store
-/.build
-/Packages
-xcuserdata/
-DerivedData/
-.swiftpm/configuration/registries.json
-.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
-.netrc
```

Comment:
Maybe this .gitignore was useful. Not sure if you should leave this one here or import the content in the root one through 🤔 
Personally I would add those in the root one too

```
xcuserdata/
*.hmap
*.ipa
*.dSYM.zip
*.dSYM
timeline.xctimeline
playground.xcworkspace
.swiftpm
.build
```